azure-pipelines.yml

```diff
@@ -1,10 +1,10 @@
 jobs:
- - template: default.yml@templates
-   parameters:
-     minrust: 1.39.0 # bytes
-     # re-enable coverage when we're on a newer version of time:
-     # https://github.com/time-rs/time/pull/265
-     # codecov_token: $(CODECOV_TOKEN_SECRET)
+  - template: default.yml@templates
+    parameters:
+      minrust: 1.42
+      # re-enable coverage when we're on a newer version of time:
+      # https://github.com/time-rs/time/pull/265
+      # codecov_token: $(CODECOV_TOKEN_SECRET)
```

Comment:
Sorry, just noticed this. Could you uncomment the `codecov_token` line and remove the comment above it now that we've bumped `time`?

Reply:
Done, looks like something else started failing in the build though...